test/features/search.feature

```diff
@@ -39,48 +40,68 @@ Feature: Search
       | edumication  |
       | dazzling     |
     And datasets:
-      | title              | publisher | author  | published | tags         | topics      | description |
-      | ooftaya Dataset 01 | Group 02  | Gabriel | Yes       | something01  | edumication | Test 01     |
-      | ooftaya Dataset 02 | Group 01  | Gabriel | Yes       | politics01   | dazzling    | Test 02     |
+      | title               | publisher | author  | published | tags         | topics      | description |
+      | DKANTest Dataset 01 | Group 02  | Gabriel | Yes       | something01  | edumication | Test 01     |
+      | DKANTest Dataset 02 | Group 01  | Gabriel | Yes       | politics01   | dazzling    | Test 02     |
+    And resources:
+      | title       | publisher | format | author | published | dataset             | description |
+      | Resource 01 | Group 01  | csv 2  | Badmin | Yes       | DKANTest Dataset 01 |             |
+      | Resource 02 | Group 01  | html 2 | Badmin | Yes       | DKANTest Dataset 02 |             |
 
+  @search_01
   Scenario: Searching datasets
     Given I am on the "Dataset Search" page
     When I search for "Dataset 01"
     Then I should be on the "Dataset Results" page
     And I should see "Dataset 01"
 
-  Scenario: See number of datasets on search page
+  @search_02
+  Scenario: See number of datasets on search page and Reset dataset search filters
     Given I am on the "Dataset Search" page
-    When I search for "ooftaya"
-    Then I should see "2" search results shown on the page
-    And I should see "2 results"
+    When I search for "DKANTest"
+    Then I should see "2 results"
+    And I should see "2" items in the "datasets" region
+    When I press "Reset"
+    Then I should see all published search content
 
-  Scenario: Filter by facet tag
+  @search_03
+  # Sites with long lists of facet items will fail unless you filter first.
+  Scenario: Filter by facets
+    # Tag
     Given I am on the "Dataset Search" page
```

Comment:
While this technically works, it's not correct to have steps other than the first in a scenario start with "Given"

See http://docs.behat.org/en/v2.5/guides/1.gherkin.html#scenarios

Reply:
done